### PR TITLE
Standalone XR support.

### PR DESCRIPTION
**Pull request by at-ninja**
Use the existing Android app support to add support for standalone XR devices, like Quests.

I added build variants to the gradle configuration to allow choosing for different projects and devices on the command line. For example,

* `./gradlew buildMobileGbufferDebug` will build an APK for a mobile phone with the vk_16_gbuffer project.
* `./gradlew buildQuestCubexrDebug` will build an APK for the Quest with the vk_04_cube_xr project.

Fixes https://github.com/google/bigwheels/issues/80

### CODE REVIEW

File: android/app/build.gradle [hunk 1]
```diff
@@ -2,20 +2,29 @@ plugins {
...
         minSdk 29
         targetSdk 32
         versionCode 1
-        versionName "1.0"
+        versionName '1.0'
 
-        testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
+        testInstrumentationRunner 'androidx.test.runner.AndroidJUnitRunner'
         externalNativeBuild {
+            ndk {
```
[the remark this answers is not in the document]
Comment (at-ninja) [reply android/app/build.gradle]: Moved this down to the Quest variant. The Quest OpenXR loader only supports `arm64` and `armeabi`.

**Comment by at-ninja** [comment]
The main value of the separate CMakeLists.txt file for this PR is overwriting the OpenXR loader with the Quest precompiled loaders. As long as that functionality is maintained, I see no reason to keep the android CMakeLists.txt around.

**Comment by Keenuts** [comment]
Hello! Created https://github.com/google/bigwheels/pull/101 to rebase this on top of #94 .
I don't have an XR device on hand, so could you just checks the sample runs fine? (it builds fine on my end, just can't run/install).

**Comment by at-ninja** [comment]
Closing in favor of #101.

**Pull request closed without merging**